src/monitoring/snapshot.rs

```diff
@@ -0,0 +1,51 @@
+use drivers::postgres;
+use postgres::rows::Row;
+
+/// struct for row of pg_stat_statements
+pub struct Snapshot {
+    pub userid: postgres::types::Oid, // OID of user who executed the statement
```

Comment:
To be honest I believe we don't need postgres types in struct. f64 and int64 would be enough